src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Yeah, true! @steida c'mon, the Este is awesome... Not a CSS master here as well (spending to much time on JS-side of the development haha), but maybe this conversation will give all of us a bit more knowledge about CSS best practices hahah :)